DEVPROD-21980 Add docs for Parsley AI (#1037)
evergreen-ci/ui: apps/parsley/docs/01-Home.md

# What is Parsley?

Parsley is Evergreen's log viewer. Parsley is designed to be a modern, user-friendly log viewer that is easy to use and navigate.

> It is specifically designed to be used to view logs from Evergreen tasks and tests, but it can also be used to view user uploaded logs.
> 
> Use the navigation on the left to learn more about Parsley and its features.

Reviewer: ```suggestion
It is specifically designed to view logs from Evergreen tasks and tests, but can also display user-uploaded logs.
```
